public/store/css/main.css

```diff
@@ -907,10 +907,13 @@ SEARCH PAGE
 	outline-offset: 0px;
 }
 
+.checkbox-control.relative {
```

Comment:
.relative is a helper class that only applies `position: relative;` so I would just remove the class in the blade, and add the `display: flex` in the block below.

Reply:
I bow to nick's css wisdom

Reply:
very well! shall do.